source/chapters/library.rst

```diff
@@ -327,12 +327,17 @@ Tracks can be loaded into a deck in several ways:
   Mixxx's waveform displays will load it into that deck.
 
 
-unLoading Tracks
+Unloading Tracks
 ==============
 
-* To unload track in Deck1, use the keyboard shortcut Shift + Left. In Deck2 you can
+Tracks can be unloaded into a deck in several ways:
+
+* To unload track in Deck 1, use the keyboard shortcut Shift + Left. In Deck 2 you can
   unload track with Shift + Right
 
+* You can use the eject track button to unload track from a deck. This button is
+  located at the bottom right, below the waveform.
```

Comment:
I would implement this now @ronso0  . Thank you so much for taking out time to review this.

Reply:
Hello @ronso0 I just implemented this, but i don't really know how to implement this `QT_SCREEN_SCALE_FACTORS=2` today is the first time I am using RST actively. The first time I used it involved no addition of pictures. What do you think about the coloured eject button? since you wanted more emphasis on it

Reply:
oh sorry, forgot you're ne wto mixxx :)
scale factors refers to running Mixxx with an user-specified scaled GUI, for example on 4k screens.
Instructions are here:
https://github.com/mixxxdj/mixxx/wiki/troubleshooting#graphical-user-interface-gui-is-too-big-or-too-small
If you don't succeed, let me know. I can upload the picture myself later today.
> What do you think about the coloured eject button?

either blur the sorroundungs or desaturate everything except the button and use a colored icon.

Reply:
> oh sorry, forgot you're ne wto mixxx :)
> scale factors refers to running Mixxx with an user-specified scaled GUI, for example on 4k screens.
> Instructions are here:
> https://github.com/mixxxdj/mixxx/wiki/troubleshooting#graphical-user-interface-gui-is-too-big-or-too-small
> If you don't succeed, let me know. I can upload the picture myself later today.
> 
> > What do you think about the coloured eject button?
> 
> either blur the surroundings or desaturate everything except the button and use a colored icon.

Hello @ronso0 I read the [instructions](https://github.com/mixxxdj/mixxx/wiki/troubleshooting#graphical-user-interface-gui-is-too-big-or-too-small) and can't seem to understand how to implement the scale factor. The instruction explains why an image maybe too big or small and how to remove scale factor from your device. can you please do it so I can learn from what you did? Thank you!

Reply:
This is what I had in mind.
![Mixxx-eject-button png](https://user-images.githubusercontent.com/5934199/95598055-bdec3480-0a4f-11eb-9b06-2dcc4762bf8a.png)
or this
![Mixxx-eject-button png_blur](https://user-images.githubusercontent.com/5934199/95598573-5f738600-0a50-11eb-90e6-63bf23dbfcab.png)

We may not need the entire deck, so feel free to crop or tweak the image further, as you like. Check what looks best in the manual context.
Then you may `git add [path to image] && git commit --amend` so it's part of your previous commit "Adding image of eject button".
`git push -f` afterwards.

Reply:
Hello @ronso0 the first Image is so nice! I would work on this now. Hoping to learn more from you! I really like the first image.

Reply:
Hello @ronso0 I just implemented this now! What do you think?

Reply:
@ronso0 I ran ```git commit -m```  before ```git --amend -m```. It was a mistake, am sorry

Reply:
Note that this PR goes into the `manual-2.2.x` branch, so the Screenshots should match the default skin used in Mixxx 2.2. We'll need to update the Screenshots later for Mixxx 2.3.

Reply:
okay, so let's leave the screenshot out of this
@hannydevelop unless you're willing to install Mixxx 2.2.4 and prepare a screenshot with the Deere skin similiar to what I posted above.
but we can also add that after #154 has been merged.
As you like..

Reply:
> okay, so let's leave the screenshot out of this
> @hannydevelop unless you're willing to install Mixxx 2.2.4 and prepare a screenshot with the Deere skin similiar to what I posted above.
> but we can also add that after #154 has been merged.
> As you like..

Ok, I would remove the Image then

Reply:
> > okay, so let's leave the screenshot out of this
> > @hannydevelop unless you're willing to install Mixxx 2.2.4 and prepare a screenshot with the Deere skin similiar to what I posted above.
> > but we can also add that after #154 has been merged.
> > As you like..
> 
> Ok, I would remove the Image then

Image has been removed

Reply:
please also remove the lonely `Unloading Tracks` header.

Reply:
Updated by current review

Reply:
Eject button image can be added later on, keyboard should be mentioned